test/e2e/fixture/certs/certs.go

```diff
@@ -14,31 +14,46 @@ import (
 func AddCustomCACert() {
 	caCertPath, err := filepath.Abs("../fixture/certs/argocd-test-ca.crt")
 	errors.CheckError(err)
-	args := []string{"cert", "add-tls", "localhost", "--from", caCertPath}
-	errors.FailOnErr(fixture.RunCli(args...))
-	args = []string{"cert", "add-tls", "127.0.0.1", "--from", caCertPath}
-	errors.FailOnErr(fixture.RunCli(args...))
+	if fixture.IsLocal() {
+		args := []string{"cert", "add-tls", "localhost", "--from", caCertPath}
```

Comment:
Could you add a comment please ? Feels a little obscure for someone not familiar with the code :D

Reply:
Done